### DISCUSSION TITLE
What theme base does OpenMage 20 use aka should I test Magento 1.9 or Magento 2.0 themes?

### DISCUSSION THREAD

**Discussion by djixas** [OpenMage/magento-lts#4182] [Questions & Answers]
Thank you, just playing around and want to know what themes to buy for what code base for magento

**Comment by kiatng** [comment]
You can take a look at https://github.com/OpenMage/magento-lts/discussions/3525

**Comment by kiatng** [comment]
For testing, use the very old rwd theme.

**Comment by pquerner** [comment]
Magento 2 Themes are not compatible. OpenMage is based on Magento 1.
